lib/cocoapods-core/specification/dsl.rb

```diff
@@ -1324,6 +1324,42 @@ def subspec(name, &block)
         subspec
       end
 
+      # The test type this specification supports. This only applies to test specifications.
+      #
+      # ---
+      #
+      # @example
+      #
+      #   test_spec.test_type = :unit
+      #
+      # @param  [Symbol] type
+      #         The test type to use.
+      attribute :test_type,
+                :types => [Symbol],
+                :multi_platform => false
+
+      # Represents a test specification for the library. Here you can place all
+      # your tests for your podspec along with the test dependencies.
+      #
+      # ---
```

Comment:
@orta @DanToml I am reading the code more I dont think `:keys` is what we want here so I removed it. Keys is meant to be used to validate the keys within a hash value, not for limiting the potential values you can have for this attribute.